src/components/ItemCard.css

```diff
@@ -8,6 +8,7 @@
   margin: 1rem;
   border: hidden;
   border-radius: 15px;
+  max-height: auto;
```

Comment:
I'm not sure if this is doing anything. I've checked the reference and they never mention `max-height: auto`. Please take a look:
https://developer.mozilla.org/en-US/docs/Web/CSS/max-height

Reply:
Not sure how and why I added this in the first place. But thank you for the link!